sessions/css-flexbox/profile/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Static Template</title>
+  </head>
+  <body>
+    <h1>My Profile</h1>
+    <main>
+      <section class="profile__header">
+        <img
+          src="https://placekitten.com/400/400"
+          alt="cute kitten"
+          class="profile__image"
+        />
+        <p>
+          Welcome to my site! I already have several years of experience in web
+          development and I am looking forward to work in new projects.
+        </p>
+      </section>
+      <hr />
+      <section>
+        <h2>Tech Stack</h2>
+        <ul class="profile__tech-stack-list" role="list">
+          <li class="profile__tech-stack-list-item">HTML</li>
+          <li class="profile__tech-stack-list-item">CSS</li>
+          <li class="profile__tech-stack-list-item">JavaScript</li>
+          <li class="profile__tech-stack-list-item">React</li>
+          <li class="profile__tech-stack-list-item">Node.js</li>
+          <li class="profile__tech-stack-list-item">MongoDB</li>
+        </ul>
+      </section>
```

Comment:
The BEM here does not make sense. We have multiple different blocks here, not just one block. I would suggest the following:
```suggestion
      <section class="profile">
        <img
          src="https://placekitten.com/400/400"
          alt="cute kitten"
          class="profile__image"
        />
        <p>
          Welcome to my site! I already have several years of experience in web
          development and I am looking forward to work in new projects.
        </p>
      </section>
      <section class="tech-stack">
        <h2>Tech Stack</h2>
        <ul class="tech-stack__list" role="list">
          <li class="tech-stack__list-item">HTML</li>
          <li class="tech-stack__list-item">CSS</li>
          <li class="tech-stack__list-item">JavaScript</li>
          <li class="tech-stack__list-item">React</li>
          <li class="tech-stack__list-item">Node.js</li>
          <li class="tech-stack__list-item">MongoDB</li>
        </ul>
      </section>
```